### ISSUE TITLE
Shimmering on returning to issue violates WCAG 2.1 criterion 2.2.2 (level A) pause, stop, hide

### ISSUE THREAD

**Issue by ChasBelov** [webcompat/webcompat.com#3847]
When I completed a report and got the choice of sending via GitHub or anonymously, I chose GitHub. After reading GitHub's terms for letting webcompat submit on my behalf, I decided I didn't want to do that and went back to the form. I got a brief message and red background that the GitHub connection had failed. However, I was then not able to submit anonymously. The report just shimmers. 

The shimmer lasts longer than five seconds and there is no way to stop it other than by leaving the page.

Steps:
1. Go to https://webcompat.com/issues/new
2. Fill out an issue
3. When I get the choice of sending via GitHub or anonymously, choose Report via GitHub
4. Log into GitHub
5. When I get the notification asking me if I want to authorize the WebCompat app, I choose to read the terms
6. I decide I don't like the terms
7. I click cancel
8. I go back to the form using my browser's back button
9. I look for a way to stop the report information section from shimmering

Expected result: There is a pause button to stop the shimmering -or- the shimmering stops after 5 seconds
Actual result: The report is shimmering and I have no way to stop it

**Comment by sgaquaakun59-png** [comment]
https://webcompat.com/issues/new
